tests/aws/app/app_test.go

```diff
@@ -63,10 +63,10 @@ func TestMain(m *testing.M) {
 	if hostIP == "" || gcpProjectID == "" {
 		log.Println("Test environments need to be setup to run server tests.")
 		flag.PrintDefaults()
-		return
-	}
-	if err := startApp(); err != nil {
-		log.Fatal("startApp: ", err)
+	} else {
+		if err := startApp(); err != nil {
+			log.Fatal("startApp: ", err)
+		}
 	}
 	os.Exit(m.Run())
```

Comment:
So `os.Exit` must be called to make the test pass? Add a comment here as a reminder.

Reply:
Done